modules/ROOT/pages/setup.adoc

```diff
@@ -35,17 +35,17 @@ If you are unsure about the Mule runtime capacity required to handle your B2B tr
 == Environment Configuration
 
 You'll need to set up and configure your environment before you begin using Partner Manager.
-Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[Configuring your Anypoint platform environments].
+Refer to the Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[Configuring your Anypoint platform environments].
 
 === Allocate Partner Manager Roles
 
-There is one Administrator role name Partner Manager Administrator. Depending on which environment you select, the user is assigned that role within the context of that environment.
+There is one Administrator role called the Partner Manager Administrator. Depending on which environment you select, the user is assigned that role within the context of that environment.
 Both roles have unrestricted access, but one role works in a Production environment and the other works in a Sandbox environment.
```

Comment:
This text is confusing because it says there is just ONE admin role, but then goes on to say "both roles have...."
Clarify:

There is only one Anypoint Partner Manager role: Partner Manager Administrator. That administrator has unrestricted access in either of two environments: the production environment or the sandbox environment.

To allocate this role:

. Navigate to either the production or sandbox Access Manager.
. Click *Partner Manager* > *Partner Manager Administrator*.

Reply:
Fixed.